twilight-model/src/gateway/presence/activity_flags.rs

```diff
@@ -92,7 +77,11 @@ mod tests {
             &ActivityFlags::EMBEDDED,
             &[Token::U64(ActivityFlags::EMBEDDED.bits())],
         );
-        // Deserialization truncates unknown bits.
-        serde_test::assert_de_tokens(&ActivityFlags::empty(), &[Token::U64(1 << 63)]);
+        // Deserialization doesn't truncate unknown bits.
+        //
+        // `bitflags` requires unsafe code to create bitflags with unknown bits.
```

Comment:
The full comment could be added here as well
```suggestion
        // Safety:
        //
        // Deserialization doesn't truncate unknown bits.
        //
        // `bitflags` requires unsafe code to create bitflags with unknown bits
        // due to an unorthodox definition of unsafe:
        //
        // <https://github.com/bitflags/bitflags/issues/262>
```